neuraxle_tensorflow/tensorflow_v2.py

```diff
@@ -199,8 +192,13 @@ def load_step(self, step: 'Tensorflow2ModelStep', context: 'ExecutionContext') -
         :return: loaded step
         """
         step.is_initialized = False
+        checkpoint_manager = tf.train.CheckpointManager(
+            step.checkpoint,
+            context.get_path(),
+            max_to_keep=3
+        )
         step.setup()
-        step.checkpoint.restore(step.checkpoint_manager.latest_checkpoint)
+        step.checkpoint.restore(checkpoint_manager.latest_checkpoint)
```

Comment:
We need to take into account the different summary ids to solve the issue because if the hyperparameters are different we should not restore. This doesn't seems fine yet for now.